content/automate/ManageIQ/AutomationManagement/AnsibleTower/Operations/Methods.class/__methods__/available_credentials.rb

```diff
@@ -25,7 +25,7 @@ def provider
               # for reconfig and retire use service
               # Or use the embedded_ansible_provider
               service = @handle.root['service_template'] || @handle.root['service']
-              service.try(:job_template, 'Provision').try(:manager) || embedded_ansible_provider
+              service.try(:playbook, 'Provision').try(:manager) || embedded_ansible_provider
```

Comment:
@lfu I have come around to agreeing with you on this change after a bit more searching.  The only model that supported a `job_template` method **that also accepts a parameter** was `ServiceTemplateAnsiblePlaybook`

With the changes to support Runner and the related refactorings, that method was renamed to `playbook`.  See https://github.com/ManageIQ/manageiq/blob/master/app/models/service_template_ansible_playbook.rb#L75

I also checked the code base at the time this `available_credentials.rb` script was initially created `git checkout 'master@{2017-02-16}'` and see that `ServiceTemplateAnsiblePlaybook` was the only model to support `job_template` with a parameter.

Reply:
👍